python/cuml/test/test_dbscan.py

```diff
@@ -60,22 +66,48 @@ def test_dbscan_predict_numpy(datatype):
                  dtype=datatype)
 
     print("Calling fit_predict")
-    cudbscan = cuDBSCAN(eps=3, min_samples=2)
+    handle, stream = get_handle(use_handle)
+    cudbscan = cuDBSCAN(handle=handle, eps=3, min_samples=2)
     cu_labels = cudbscan.fit_predict(gdf)
     skdbscan = skDBSCAN(eps=3, min_samples=2)
     sk_labels = skdbscan.fit_predict(X)
     print(X.shape[0])
     for i in range(X.shape[0]):
         assert cu_labels[i] == sk_labels[i]
+    if stream is not None:
+        stream.sync()
```

Comment:
To my understanding a sync is needed in any case. See my comment above.

Reply:
This is done already.